Plugin/View/LayoutPlugin.php

```diff
@@ -70,13 +70,16 @@ public function __construct(
      * @param  $blockId
      * @return string
      */
-    protected function injectHtmlAttribute($html, $blockId)
+    protected function injectHtmlAttribute($html, $blockId, $stack)
     {
         if (!$html || !$this->config->canInjectCode()) {
             return $html;
         }
 
-        $html = '<!-- MSPDEVTOOLS[' . $blockId . '] -->' . $html . '<!-- /MSPDEVTOOLS[' . $blockId . '] -->';
+
+        $html = '<!-- START[' . $blockId . '] - ' . end($stack) . ' -->' . $html . '<!-- /END[' . $blockId . '] - ' . end($stack) . ' -->';
```

Comment:
@phoenix128 

The reason I changed it to START and END is that it is shorter but is it possible to change it to MSPDEV or as you set MSPDEVTOOLS_START but it is better to set it to START_MSPDEVTOOLS for the better overview in the broncode